packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -449,7 +449,12 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
     }
   }
 
-  void _checkStart(Duration timestamp, int pointer) {
+// ignore: public_member_api_docs
+//
+// Exposed to test for null timestamp.
```

Comment:
fly-by comment: please don't do this. Anything that is public needs docs.

Reply:
(ideally, we would find a better way to test this without making it public, if we can't it needs real docs because dartdoc will process it and publish the method to our docs)